### ISSUE TITLE
Had to include Sensor Map for it to work.

### ISSUE THREAD

**Issue by cmplant3210** [gjr80/weewx-bloomsky#1]
Hello,

  I had to add the following into the Bloomsky stanza in weewx.conf for the driver to work.  Is that something you can add in to your release so that it automatically goes in for others that use your driver?  

  [[sensor_map]]
        deviceID = DeviceID
        deviceName = DeviceName
        [[[Data]]]
            outTemp = Temperature
            pressure = Pressure
            txBatteryStatus = Voltage
            UV = UVIndex
            outHumidity = Humidity
            imageURL = ImageURL
            deviceType = DeviceType
            night = Night
            imageTimestamp = ImageTS
        [[[Point]]]
            inTemp = Temperature
            inHumidity = Humidity
        [[[Storm]]]
            rainRate = RainRate
            windSpeed = SustainedWindSpeed
            windDir = WindDirection
            windGust = WindGust
            dailyRain = RainDaily

**Comment by gjr80** [comment]
That is interesting as the driver already includes that mapping as the default map, in other words if you do not specify a `sensor_map` in `weewx.conf` the default is used. As part of the WeeWX start up the driver logs the map will be used. Do you still have the logs from when you started WeeWX without any `sensor_map` in `weewx.conf`? If so I would like to see the WeeWX startup from when WeeWX was started through until the main loop starts. If you don't still have the logs can you reproduce the issue and provide the requested log output?

thanks

**Comment by cmplant3210** [comment]
Now that you mention it, I believe I was only having issues getting pressure to work.  I'll have to setup another instance of weewx with your driver, but I will let you know exactly what the issue was.

**Comment by gjr80** [comment]
OK, if it was only a pressure problem (as opposed to nothing working) don't go to too much trouble, I will see if I can get a VM running in next day or so with a clean install of the extension to see if I come across any issues. If I don't I'll let you know and see what you can do.

**Comment by cmplant3210** [comment]
I got a VM running, and yeah, its just the barometer thats not reading anything.  The Temp and Humidity are working.  I changed the weewx.conf logging to 1 but its not showing any errors, its just running through normally.  If you need logs, where exactly should i pull them from?

**Comment by gjr80** [comment]
Well I just started up a VM with WeeWX 3.8.2 and a clean Bloomsky driver install. As soon as WeeWX started the error was plain, `barometer` was not showing in the sensor map in the log:
```
Jan 31 23:24:50 stretch21 weewx[6281]: bloomsky: MainThread: sensor map is {'deviceName': 'DeviceName', 'Data': {'raining': 'Rain', 'luminance': 'Luminance', 'barometer': 'Pressure', 'outHumidity': 'Humidity', 'imageTimestamp': 'ImageTS', 'txBatteryStatus': 'Voltage', 'imageURL': 'ImageURL', 'UV': 'UVIndex', 'outTemp': 'Temperature', 'deviceType': 'DeviceType', 'night': 'Night'}, 'deviceID': 'DeviceID', 'Storm': {'windDir': 'WindDirection', 'rainDaily': 'RainDaily', 'windSpeed': 'SustainedWindSpeed', 'rainRate': 'RainRate', 'windGust': 'WindGust'}, 'Point': {'inHumidity': 'Humidity', 'inTemp': 'Temperature'}}
```

Had a look at the default sensor map in the driver and for some reason `barometer` is not included. I have no idea why, it could be because I was still looking into whether the Bloomsky `Pressure` field is equivalent to the WeeWX `barometer`, `pressure` or `altimeter` field. More likely I overlooked it.

I have put together v0.1.1 of the driver that includes `barometer` in the sensor map but it seems I have introduced another issue. Since the driver will work as long as `barometer` is included in a sensor map in `weewx.conf` I will leave this for now and release the v0.1.1 driver next week when I can sort out this other issue I have introduced.

Thanks for pointing this one out.

**Comment by gjr80** [comment]
Fixed vide commit 8160b8c939fa88092689e24dc7abedff893482fd but never released.
Release 0.1.1 includes this fix.
